xml/upnp_banners.xml

```diff
@@ -531,6 +545,38 @@
     <param pos="1" name="os.version"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:apple:mac_os_x:{os.version}"/>
   </fingerprint>
+  <fingerprint pattern="^Loxone Miniserver (.*) UPnP/1.0$">
+    <description>Loxone Miniserver Smart Home</description>
+    <example host.name="some name">Loxone Miniserver some name UPnP/1.0</example>
+    <param pos="0" name="hw.vendor" value="Loxone"/>
+    <param pos="0" name="hw.product" value="Miniserver"/>
+    <param pos="0" name="hw.device" value="Management"/>
+    <param pos="1" name="host.name"/>
+  </fingerprint>
+  <fingerprint pattern="^RouterOS/(\S+)UPnP/1.0 MikroTik UPnP/1.0$">
+    <description>Mikrotik RouterOS</description>
+    <example os.version="6.43">RouterOS/6.43UPnP/1.0 MikroTik UPnP/1.0</example>
+    <param pos="0" name="os.vendor" value="MikroTik"/>
+    <param pos="0" name="os.device" value="Router"/>
+    <param pos="0" name="os.family" value="RouterOS"/>
+    <param pos="0" name="os.product" value="RouterOS"/>
+    <param pos="1" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="^Roku UPnP/\S+ Roku/(\S+)$">
+    <description>Roku with a version</description>
+    <example hw.version="8.1.6">Roku UPnP/1.0 Roku/8.1.6</example>
+    <param pos="0" name="hw.vendor" value="Roku"/>
+    <param pos="0" name="hw.product" value="Roku"/>
+    <param pos="0" name="hw.device" value="Media Server"/>
+    <param pos="1" name="hw.version"/>
+  </fingerprint>
+  <fingerprint pattern="^Roku UPnP/\S+ MiniUPnPd/\S+$">
+    <description>Roku without a version</description>
+    <example>Roku UPnP/1.0 MiniUPnPd/1.4</example>
+    <param pos="0" name="hw.vendor" value="Roku"/>
+    <param pos="0" name="hw.product" value="Roku"/>
+    <param pos="0" name="hw.device" value="Media Server"/>
```

Comment:
I'm torn on the use of `Media Server` here.  It's just a player much like a Chromecast but perhaps we lump all such devices into this category to prevent label sprawl.

Reply:
Yes, my thinking was this devices like this serve media, and the fact that some play it vs others store and play it is an important distinction but one that it is outside recog currently, IMO :)